libkbfs/cr_chains.go

```diff
@@ -864,6 +858,7 @@ func (ccs *crChains) getPaths(ctx context.Context, blocks *folderBlockOps,
 		for _, op := range chain.ops {
 			op.setFinalPath(p)
 		}
+		paths = append(paths, p)
```

Comment:
Just checking: this line presumably moved so that a path is only added if a chain for it exists. Is that correct?